data/remnant/remnant 2 side missions.txt

```diff
@@ -605,7 +605,7 @@ mission "Remnant: Shattered Light 6"
 		has "Remnant: Shattered Light 5: done"
 	on offer
 		conversation
-			`On <destination> the spaceport is awash in conversation, and you spot snippets here and there of people talking about the recovery of the Shattered Light. Many are speculating as to what happened to it in the first place, while others are discussing the technical merits of what the research team did to recover it. Details travel quickly, it seems.`
+			`On <origin> the spaceport is awash in conversation, and you spot snippets here and there of people talking about the recovery of the Shattered Light. Many are speculating as to what happened to it in the first place, while others are discussing the technical merits of what the research team did to recover it. Details travel quickly, it seems.`
```

Comment:
```suggestion
			`On <origin>, the spaceport is awash in conversation, and you spot snippets here and there of people talking about the recovery of the Shattered Light. Many are speculating as to what happened to it in the first place, while others are discussing the technical merits of what the research team did to recover it. Details travel quickly, it seems.`
```

Reply:
Yes, there should be a comma.

Reply:
I'm dubious about the comma.

It's "On Caelian the spaceport is awash with talk", not "On Caelian" pause "the spaceport is awash with talk.

I'm not saying that having the comma there is *wrong*, just it wasn't the flow of the sentence that I intended.

Having the comma there makes it feel like it's something more along the lines of:

"Meanwhile on Caelian... The spaceport is awash in conversation." 

To me it distances the reader from the event, whereas it's supposed to be the reader plunged into it. There's no pause for breath or distance observation, the player is there in the midst of it, actively happening.

Reply:
Written rules for commas do not necessarily coincide with the pacing of speech if that passage were read aloud. The coincidence between the two is just a convenient short hand which -usually- holds as valid. Often, the placement of commas in written text is more about the grouping of information, and it is indeed most correct to include the comma here for the introductory prepositional phrase.

Preserving the feel you want would be better served (and more grammatically correct) moving that phrase, switching the word order to 'The spaceport on Caelian is awash in conversation'.

That, of course, though, loses the impact of the emphasis on the location, because that emphasis on the specific information is exactly why it's set aside by a comma in the first place.

To use your own comparison, it's less "Meanwhile... on Caelian" and more "-on Caelian, specifically, which is important to draw attention to even if already implied by (or even explicit in) the rest of the general text-," versus the revision suggested above being "which, in case you didn't already know, is on Caelian."

Reply:
OK, thanks for the explanation. I'm good with the comma for this.